src/html/scope.rs

```diff
@@ -9,10 +9,14 @@ use stdweb::web::{Element, Node};
 /// Holder for the element.
 pub type NodeCell = Rc<RefCell<Option<Node>>>;
 
+pub(crate) enum ManyOrSingleMessage<COMP: Component> {
+    Single(COMP::Message),
+    Many(Vec<COMP::Message>),
+}
 /// Updates for a `Components` instance. Used by scope sender.
 pub(crate) enum ComponentUpdate<COMP: Component> {
     /// Wraps messages for a component.
-    Message(COMP::Message),
+    Message(ManyOrSingleMessage<COMP>),
```

Comment:
I think one enum is enough. Let's add `ComponentUpdate::MessageBatch` and remove `ManyOrSingleMessage` enum